policy/bundle_s3_resolver.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package policy
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+
+	awsconfigv2 "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/rs/zerolog/log"
+)
+
+type s3Resolver struct{}
+
+func (l *s3Resolver) Load(ctx context.Context, path string) (*Bundle, error) {
+	cfg, err := awsconfigv2.LoadDefaultConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	bucket := strings.TrimPrefix(path, "s3://")
+	s3Client := s3.NewFromConfig(cfg)
+	files, err := s3Client.ListObjects(ctx, &s3.ListObjectsInput{Bucket: &bucket})
+	if err != nil {
+		return nil, err
+	}
+	mergedBundle := &Bundle{}
+	resolvedNames := []string{}
```

Comment:
How can we pull just one key? like `https://mybucket.s3.amazonaws.com/policy.mql.yaml`

Reply:
Great point, that was not implemented (yet). I added this in with the new commit.